m4_sessions/1711-inning/Group_4.md

```diff
@@ -1,9 +1,25 @@
 Group Four
 
-Session Title: Description: Why should students come to this session?
+Session Title: How to write a kicka$$ technical blog post
+
+Description: Why should students come to this session?
```

Comment:
I'm really interested in how this will go.  The "Cold outreach" session last inning was really awesome and you can look at it here: https://github.com/turingschool/gear-up/blob/master/m4_sessions/1710-inning/crushing_cold_outreach.md